bsp4kt/BUILD

```diff
@@ -2,7 +2,7 @@ load("@rules_jvm_external//:kt_defs.bzl", "kt_jvm_export")
 
 kt_jvm_export(
     name = "bsp4kt",
-    srcs = glob(["src/**/*.kt"]),
+    srcs = glob(["src/**/*.kt"]) + glob(["resources/**/*.kt"]),
```

Comment:
a te resources to czemu jako srcs a nie resources?

Reply:
Wtedy nie da się łatwo używać rzeczy tam zdefiniowanych (a przynajmniej mi się nie udało tego zrobić).
Nie potrafiłam też zdefiniować tego bezpośrednio w src bo cały ten folder jest nadpisywany przy generacji, a nie chciałam grzebać już w tym skrypcie kopiującym.
Alternatywnie można jeszcze zrobić tak jak ty to zrobiłeś z `Preconditions.java`, żeby zdefiniować plik jako resources w `codegen` i go "kopiować" przy generacji